README.md

```diff
@@ -5,3 +5,183 @@ UCL API Main Repository
 We're an amazing project, so obviously we have tests :sparkles:  
 Make sure you have the requirements installed in your virtual environment, and then run:  
 `python manage.py test --testrunner 'roombookings.custom_test_runner.NoDbTestRunner'`
+
+## Setting Up and Building
+This will walk you through setting yourself up a local development environment. This is different to if you want to deploy this into production, but luckily we have an internal set of installation scripts for this. Therefore, this guide will not walk you through, for example, getting Gunicorn or Nginx running. This is simply getting an environment up that can be used to develop the API. Testing with Nginx and Gunicorn is done in staging prior to a production deployment.
+
+### Requirements
```

Comment:
This is from like a server perspective, right?

I think it would definitely make sense to include Mac/Windows specific development instructions here.